src/main/java/com/prgrms/springbootjpa/domain/Customer.java

```diff
@@ -0,0 +1,18 @@
+package com.prgrms.springbootjpa.domain;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "customers")
+public class Customer {
+    @Id
+    private Long id;
+    private String firstName;
+    private String lastName;
```

Comment:
자바와 db의 컨벤션에 맞춰 스네이크 케이스로 필드와 컬럼명을 매핑하면 어떨까요?
```suggestion
    @Column(name="last_name")
    private String lastName;
```